apps/builder/css-tree-list.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "css-tree/utils" {
```

Comment:
Please define this type in the root definition

Reply:
Or wait, why not just import from css-tree?

Reply:
Interesting, looks like it's just the IDE that kept complaining that `List` is missing from export of `css-tree`.

Looks like this file https://github.com/webstudio-is/webstudio-builder/blob/main/%40types/css-tree.d.ts
`css-tree.d.ts` from the root of the project needs a update 👍 
https://unpkg.com/browse/@types/css-tree@2.3.1/index.d.ts

Can this be moved to the. package.json instead of a custom `css-tree.d.ts` file ? Was there some specific issue so the typings are maintained separately in repo

Reply:
Probably types [@types/css-tree@2.3.1/index.d.ts](https://unpkg.com/browse/@types/css-tree@2.3.1/index.d.ts) here now is good enough to use.
At the moment local d.ts was created `lexer` wasn't available